atc/atccmd/command.go

```diff
@@ -1071,6 +1073,15 @@ func (cmd *RunCommand) backendComponents(
 		cmd.ResourceWithWebhookCheckingInterval = cmd.ResourceCheckingInterval
 	}
 
+	if cmd.ResourceTypeCheckingInterval < cmd.ResourceCheckingInterval {
```

Comment:
I think this is also applicable like the webhook code block above, in my experience resource types take longer to get than resources so.